.github/workflows/publishing.yml

```diff
@@ -279,6 +279,23 @@ jobs:
           TAGS: ${{ steps.metadata.outputs.tags }}
         run: cosign sign --yes --recursive "${TAGS}@${DIGEST}"
 
+      - name: Sign image with NGC CLI
+        if: steps.release_info.outputs.ngc_registry
+        env:
+          TAGS: ${{ steps.metadata.outputs.tags }}
+          NGC_CLI_API_KEY: ${{ secrets.NGC_CREDENTIALS }}
+          NGC_CLI_ORG: ${{ github.repository_owner }}
+          NGC_CLI_TEAM: 'nightly'
+        run: |
+          wget -O ngccli_linux.zip --content-disposition \
+            https://api.ngc.nvidia.com/v2/resources/nvidia/ngc-apps/ngc_cli/versions/3.28.0/files/ngccli_linux.zip
+          if [ -z "$(sha256sum ngccli_linux.zip | grep -o '74f469a2082e90a4561f09eb0d483b67044e91be727066122253436c140a7b54 ')" ]; then
```

Comment:
Is there another way to test if the file is signed other than using a magical SHA value?

Reply:
Well, you are supposed to check that the sha is actually the correct one to know that you downloaded what you intended to download. This is what the official documentation recommends you do unfortunately. The sha is documented in the download page, and it won't break since we fix the version number. When we update to a new version, this needs to be updated according to the docs for the new version.